src/desktop/public/.well-known/assetlinks.json

```diff
@@ -0,0 +1,9 @@
+[{
+  "relation": ["delegate_permission/common.handle_all_urls"],
+  "target": {
+    "namespace": "android_app",
+    "package_name": "net.artsy.app",
+    "sha256_cert_fingerprints":
+    ["FA:C6:17:45:DC:09:03:78:6F:B9:ED:E6:2A:96:2B:39:9F:73:48:F0:BB:6F:89:9B:83:32:66:75:91:03:3B:9C"]
```

Comment:
quick question, where are all these coming from?

Reply:
@pvinis I think the sha256 fingerprint there is a debug one generated by android studio for this project. Before releasing this for production use, there is need to add the sha256 fingerprint from google play store.